File: problem-solving/programmers/level01/두_정수_사이의_합.py
# https://programmers.co.kr/learn/courses/30/lessons/12912

def solution(a, b):
    answer, start, end = 0, 0, 0
    if a>b:
        start, end = b, a
    else:
        start, end = a, b
    for i in range(start, end+1):
        answer += i
    return answer

# if __name__ == "__main__":
#     print(solution(3,5))
#     print(solution(3,3))
#     print(solution(5,3))

File: problem-solving/programmers/level01/두_정수_사이의_합.java
// https://programmers.co.kr/learn/courses/30/lessons/12912
class Solution {
    public long solution(int a, int b) {
        long answer = 0;
        if (a < b) {
            for (int i=a; i<=b; i++) {
                answer += i;
            }
        } else {
            for (int i=b; i<=a; i++) {
                answer += i;
            }
        }
        return answer;
    }
}